src/main/webapp/WEB-INF/tags/instructor/home/coursePanel.tag

```diff
@@ -15,9 +16,30 @@
             <div class="mobile-margin-top-10px col-sm-6">
                 <span class="mobile-no-pull pull-right">
                     <c:forEach items="${courseTable.buttons}" var="button">
-                        <a data-toggle="tooltip" data-placement="top" ${button.attributesToString}>
-                            ${button.content}
-                        </a>
+                        <c:choose>
+                            <c:when test="${fn:length(button.nestedElements) gt 0}">
+                                <div class="dropdown courses-table-dropdown">
+                                    <button class="btn btn-primary btn-xs dropdown-toggle" type="button" data-toggle="dropdown">
```

Comment:
The difference between this and the one we have on the "remind" button that gives it the seperator is because it is done via a ```btn-group``` class

![Uploading Screen Shot 2016-09-05 at 3.46.54 PM.png…]()